Ical.Net/Evaluation/PeriodListEvaluator.cs

```diff
@@ -19,16 +19,10 @@ public PeriodListEvaluator(PeriodList rdt)
         _mPeriodList = rdt;
     }
 
-    public override IEnumerable<Period> Evaluate(CalDateTime referenceDate, CalDateTime? periodStart, CalDateTime? periodEnd, bool includeReferenceDateInResults)
+    public override IEnumerable<Period> Evaluate(CalDateTime referenceDate, CalDateTime? periodStart, CalDateTime? periodEnd, EvaluationOptions options)
     {
```

Comment:
No coverage, wondering when this method would get called

Reply:
Within Ical.Net not at all I assume, because the only place where it *could* be used is `RecurringEvaluator`, where we use `Recurrable.RecurrenceDates.GetAllPeriodsByKind()`. Remove?!